fe/src/App.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-import { Home } from './components/pages/Home/index';
 import { About } from './components/pages/About/index';
 import { Error } from './components/pages/Error/index';
+import { Home } from './components/pages/Home/index';
```

Comment:
Whats the purpose of this change? Look it in the diff.

Reply:
I have this thing on that auto organizes imports and that's what happened here.